raiden/api/rest.py

```diff
@@ -561,7 +561,19 @@ def open(
             token_address=to_checksum_address(token_address),
             settle_timeout=settle_timeout,
         )
+        token = self.raiden_api.raiden.chain.token(token_address)
+        balance = token.balance_of(self.raiden_api.raiden.address)
         try:
+            if total_deposit is not None and total_deposit > balance:
```

Comment:
as I said before please remove all of these additions from inside the `try:` clause. Move them back up where `balance` is.